pkg/transportrequest/gitutils_test.go

```diff
@@ -198,9 +198,34 @@ func TestRetrieveLabelStraightForward(t *testing.T) {
 				}
 			}
 		})
+		t.Run("default label with default reg ex", func(t *testing.T) {
+			commitIter := &commitIteratorMock{
+				commits: []object.Commit{
+					object.Commit{
+						Hash:    plumbing.NewHash("3434343434343434343434343434343434343434"),
+						Message: "TransportRequest: 12345678",
+					},
+				},
+			}
+			labels, err := FindLabelsInCommits(commitIter, "TransportRequest\\s?:")
```

Comment:
🆗 With the old regex we still get the label.

Reply:
🆗